### ISSUE TITLE
evaluation

### ISSUE THREAD

**Issue by knelk** [HKUST-Aerial-Robotics/ESVO#29]
Hi, thanks for the amazing work.

Can you please clarify how do you compute your error metrics, exactly?

Tab. V shows the relative error. Do you use the RPE-script from https://cvg.cit.tum.de/data/datasets/rgbd-dataset/tools#evaluation? If yes, please specify how you call the script (which parameters), but mainly how do you set --max_pairs?

Also, do you do any post-/preprocessing to the trajectory, e.g. interpolation,...?

**Comment by knelk** [comment]
since you report the units [°/s] and [cm/s], I assume you use the script with ``fixed_delta``, i.e. the following way:

````
python scripts/evaluate_rpe.py {fnamegt} {fnameest} --fixed_delta --delta 1 --delta_unit s
````

Is that correct?

**Comment by Ethan-Zhou** [comment]
Yes, I cannot remember the exact command, but it makes sense to me.
